Bottom: 77505b8e184506d1b1f3d1e18e877a1917c37cfd
Top:    bf84bdc02516c71cfd142f63cb545ea4ae880e73
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:13 +0100

Cure crosshair lagging on screen

FIXME: SHOULD NOT RE-WRITE THE GTK HID


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 83f09b7..67781a3 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -675,9 +675,7 @@ ghid_port_window_motion_cb (GtkWidget * widget,
   queue_tooltip_update (out);
 #endif
 
-  ghid_show_crosshair (FALSE);
-  ghid_show_crosshair (TRUE);
-  if (moved && have_crosshair_attachments ())
+  if (moved)
     ghid_draw_area_update (gport, NULL);
   return FALSE;
 }
